backend/tests/test_watchlist_router.py

```
@@ -10,6 +10,35 @@
 from __future__ import annotations
 
 import pytest
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.pool import StaticPool
+
+from app.models.database import Base, WatchlistTicker
+from app.services.watchlist import add_ticker, list_watchlist
+
+
+@pytest.fixture()
```

Ruff (0.15.15)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)